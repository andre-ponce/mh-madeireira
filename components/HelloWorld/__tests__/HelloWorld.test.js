import React from 'react';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';

import HelloWorld from '../HelloWorld';

describe('Hello World component', () => {
  describe('With Enzyme', () => {
    it('It must contain the text "Hello World" inside an H1 in the Hello World component', () => {
      const wrap = shallow(<HelloWorld />);

      expect(wrap.find('h1').text()).toEqual('Hello World');
    });
  });

  describe('With Snapshot Testing', () => {
    it('should render Hello World', () => {
      const component = (
        <HelloWorld />
      );

      const tree = renderer.create(component).toJSON();

      expect(tree).toMatchSnapshot();
    });
  });
});
