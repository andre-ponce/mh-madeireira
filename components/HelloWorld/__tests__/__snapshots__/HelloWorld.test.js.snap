// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hello World component With Snapshot Testing should render Hello World 1`] = `
<h1>
  Hello World
</h1>
`;
